examples/docs_snippets/docs_snippets/guides/dg/migrating-definitions/2-definitions-before.py

```diff
@@ -16,5 +17,5 @@
         jobs=[sync_tables_job, regenerate_analytics_job],
         schedules=[sync_tables_daily_schedule, regenerate_analytics_hourly_schedule],
     ),
-    dg.components.load_defs(my_existing_project.defs),
```

Comment:
as a norm I think we should use kwargs (and probably require them) to make this more self-documenting

Reply:
mandated kwargs